src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/Converters/IAppIntentJsonConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using MorganStanley.Fdc3;
+using AppIntent = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.AppIntent;
+
+namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Converters;
+
+public class IAppIntentJsonConverter : JsonConverter<IAppIntent>
```

Comment:
class IAppIntent...?

Reply:
Yeah, I haven't know how to name it I'll rename all of the related classes. :)